dashboard/app/models/census/README.md

```diff
@@ -10,29 +10,87 @@ The different forms are:
 
 ## College Board data about which schools offer AP Computer Science
 
 This data is stored in `ap_cs_offerings` and is imported from S3 as part of database seeding.
 
+The `ap_school_codes` table is used to map between NCES school ids and college board school codes.
+
+[bin/oneoff/census/ap_data](https://github.com/code-dot-org/code-dot-org/tree/staging/bin/oneoff/census/ap-data) has several scripts that were used to work with the AP data.
+
 ## International Baccalaureate data about which schools teach IB Computer Science
 
 This data is stored in the `ib_cs_offerings` table.
 
+The `ib_school_codes` table is used to map between NCES school ids and International Baccalaureate school codes.
+
+[bin/oneoff/census/ib_data](https://github.com/code-dot-org/code-dot-org/tree/staging/bin/oneoff/census/ib-data) has scripts that were used to work with the IB data.
+
 ## State-level course data
 
 We get data from some states that list which schools teach which Computer Science classes. This data is stored in the `state_cs_offerings` table.
 
+[bin/oneoff/census/state_data](https://github.com/code-dot-org/code-dot-org/tree/staging/bin/oneoff/census/state-data) has scripts that were used to work with state data.
+
+## Inaccuracy Investigations and Overrides
+
+If somebody submits a survey indicating that they think the data we have is wrong then that reported inaccuracy will appear for review at https://studio.code.org/census/review (you need to have `census_reviewer` permissions to view that page.) After reviewing the report, a row will be created in `census_inaccuracy_investigations`. The reviewer can choose to override the summary for the school as part of the review process. In that case we will also create a row in the `census_overrides` table. That row will be referenced by the `census_inaccuracy_investigations` row.
+
+See also:
+* [The controller for the review page](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/census_reviewers_controller.rb)
+* [The react component for the  review page](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/census2017/CensusInaccuracyReview.jsx)
+
 # Summarization
 
 Each school may have multiple data points for the same school year. All of that data is aggregated into a simple summary that says if we believe the school teaches/taught Computer Science in a given year. The possible values are:
 
 * **Yes** - We are fairly confident that the school teaches CS
 * **No** - We are fairly confident that the school **does not** teach CS
 * **Maybe** - The data did not give us a strong indication one way or the other
+* **Historical Yes** - In one of the past two years we were fairly confident that the school taught CS
```

Comment:
What exactly differentiates Yes vs. Historical Yes?

Reply:
Oh, Historical Yes is when we don't have any any data for the current year?

Reply:
Yes: https://github.com/code-dot-org/code-dot-org/pull/21464/files/d6aebb35df432c9dd984e983cb52ffc2c1eece3a#diff-ff74bc0f76cb06c0a9eb730e3ff9b416R60

Reply:
I updated the text to make this clearer.